tests/conftest.py

```diff
@@ -16,17 +16,85 @@
 import os
 from unittest import mock
 
+import boto3
 import pytest
 import tfworker
 import tfworker.commands.base
 import tfworker.commands.root
 import tfworker.providers
+from moto import mock_dynamodb2, mock_s3, mock_sts
 from pytest_lazyfixture import lazy_fixture
-from tfworker.util.copier import Copier, CopyFactory
 
 
-@pytest.fixture
-def rootc():
+@pytest.fixture(scope="class")
+def aws_credentials():
+    """Mocked AWS Credentials for moto."""
+    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
+    os.environ["AWS_SECRET_ACCESS_KEY"] = "testing"
+    os.environ["AWS_SECURITY_TOKEN"] = "testing"
+    os.environ["AWS_SESSION_TOKEN"] = "testing"
+
+
+@pytest.fixture(scope="class")
+def s3_client(aws_credentials):
+    with mock_s3():
+        boto3.setup_default_session()
+        yield boto3.client("s3", region_name="us-west-2")
+
+
+@pytest.fixture(scope="class")
+def dynamodb_client(aws_credentials):
+    with mock_dynamodb2():
+        # boto3.setup_default_session()
```

Comment:
Why isn't this needed here?

Reply:
before using the mocked aws authenticator I was trying to troubleshoot, good catch this is actually no longer needed anywhere.